analysers/analyser_merge_power_branch_FR.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Frédéric Rodrigo 2017                                      ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from modules.OsmoseTranslation import T_
+from .Analyser_Merge import Analyser_Merge, SourceOpenDataSoft, CSV, Load, Conflate, Select, Mapping
+
+
+class Analyser_Merge_Power_Substation_FR(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.def_class_missing_osm(item = 7190, id = 22, level = 3, tags = ['merge', 'power', 'fix:chair'],
+            title = T_('Power transmission branch not known from operator'),
+            detail = T_('This power transmission branch from OSM is now known to operator. It may have been destroyed or moved.'))
+        self.def_class_possible_merge(item = 8281, id = 23, level = 3, tags = ['merge', 'power', 'fix:chair'],
+            title = T_('Power transmission branch, integration suggestion'),
+            detail = T_('This existing power transmission branch can be integrated with official attributes.'))
```

Comment:
```suggestion
            title = T_('Power transmission branch, integration suggestion')),
```

There is already generic explanation add to all merge items.

https://github.com/osm-fr/osmose-backend/blob/master/analysers/Analyser_Merge.py#L1050-L1093

When I ask about commons words, it was about "transmission branch".

Reply:
I've replaced transmission branch by connection in title, despite there is no better wording to me